### PR TITLE
Ch 74 fix svgwave responsiveness in about page

### PR DESCRIPTION
**Pull request by DarrickFauvel**
**Files:**
About.jsx

**Problem:**
Wave image had space gap appear between the top and bottom waves at smaller screen sizes. 
General page responsiveness needs improvement.

**Solution:**
Removed grid-gap and better positioned the waves to top/bottom of project section and team articles.

### CODE REVIEW

**Review by Greimil — APPROVED** [review]

File: src/pages/About.jsx [hunk 1]
```diff
@@ -1,152 +1,145 @@
...
 
-          <SvgComponent className="absolute -top-8 left-0" />
-          <SvgComponent className="absolute top-0 left-0 rotate-180" />
-        </section>
-      </div>
+        {/* <SvgComponent className="absolute -top-8 left-0" />
+          <SvgComponent className="absolute top-0 left-0 rotate-180" /> */}
+        <SvgComponent className="absolute -top-1 left-0 rotate-180" />
+      </section>
+      {/* END Team section */}
```
Comment (timDeHof) [inline src/pages/About.jsx]: Svg components doesn't have a color associated with them. You can add `fill-{color}` to color them. I believe the color of the waves on the homepage is #FFC945.

**Review by timDeHof — CHANGES_REQUESTED** [review]
The svg wave don't have color on them. If their color is the same as the svgwaves on the homepage then add`fill-[#FFC945]` to their classnames.

File: src/pages/About.jsx (hunk 1 above)
Comment (DarrickFauvel) [reply src/pages/About.jsx]: Thanks, Tim. I added the class `fill-[#FFC945]` to each of the wave components.

**Review by joekotvas — APPROVED** [review]
Looks good.

**Review by timDeHof — APPROVED** [review]

**Pull request merged**